protocol/shannon/mode_delegated.go

```diff
@@ -20,21 +20,39 @@ import (
 // https://www.notion.so/buildwithgrove/Different-Modes-of-Operation-PATH-LocalNet-Discussions-122a36edfff6805e9090c9a14f72f3b5?pvs=4#122a36edfff680eea2fbd46c7696d845
 
 // getDelegatedGatewayModeApps returns the set of permitted apps under Delegated gateway mode, for the supplied HTTP request.
-func (p *Protocol) getDelegatedGatewayModeApps(_ context.Context, httpReq *http.Request) ([]*apptypes.Application, error) {
+func (p *Protocol) getDelegatedGatewayModeApps(ctx context.Context, httpReq *http.Request) ([]*apptypes.Application, error) {
+	logger := p.logger.With(
+		"gateway_mode", p.gatewayMode,
+		"gateway_addr", p.gatewayAddr,
```

Comment:
Are you sure we want to add this to every single log for every request? Seems like it might add a lot of noise. WDYT of just adding these to the debug logs? Not a strong opinion, just curious.